### DISCUSSION TITLE
vicuna-13b-4bit-128g.safetensors' is not a valid JSON file

### DISCUSSION THREAD

**Discussion by enorrmann** [oobabooga/text-generation-webui#1031] [Q&A]
after following the installation steps for linux and trying to start with 
 python server.py
i get

CUDA SETUP: Loading binary /home/emilio/miniconda3/envs/textgen/lib/python3.10/site-packages/bitsandbytes/libbitsandbytes_cuda113.so...
Loading vicuna-13b-4bit-128g.safetensors...
OSError: It looks like the config file at 'models/vicuna-13b-4bit-128g.safetensors' is not a valid JSON file.

**Comment by tnsai** [comment]
You have to download the entire model repository.  E.g. grab from TheBloke at Huggingface by:

```
cd text-generation-webui
python download-model.py TheBloke/vicuna-13B-1.1-GPTQ-4bit-128g
```

Don't forget to set the environment variables HF_USER and HF_PASS with your HuggingFace username and password before running the above.
